wrapper/src/main/java/software/amazon/jdbc/plugin/strategy/fastestresponse/NodeResponseTimeMonitor.java

```diff
@@ -85,55 +84,38 @@ public NodeResponseTimeMonitor(
 
     // Report current response time (in milliseconds) to telemetry engine.
     // Report -1 if response time couldn't be measured.
-    this.responseTimeMsGauge = this.telemetryFactory.createGauge(
+    // TODO: this is not used anywhere, should we remove it?
```

Comment:
Should we remove this gauge or add logic to actually use it (it is currently unused)?

Reply:
The gauge is initialized and call provided lambda to get values and report them. It's in use.

Reply:
Ah okay, sounds like the callback is used internally, didn't realize